runtime/redirector/redirector.c

```diff
@@ -463,6 +464,62 @@ parseMemorySizeValue(char *option)
 	return result;
 }
 
+#define GC_POLICY_OPTION "-Xgcpolicy:"
+#define LENGTH_GC_POLICY_OPTION (sizeof(GC_POLICY_OPTION) - 1)
+
+static void
+checkEnvOptions(char *envOptions, int *gcPolicy, char **xcompressedstr, char **xnocompressedstr, char **xjvmstr, int *xjvm, char **namedVM, size_t *nameLength, char **xmxstr)
+{
+	char *gcPolicyString = NULL;
+
+	gcPolicyString = findStartOfMostRightOption(envOptions, GC_POLICY_OPTION);
```

Comment:
Why not fold this initialization into the declaration?

Reply:
Updated.